.github/workflows/on_daily_with_change.yml

```diff
@@ -0,0 +1,28 @@
+# Kick off a daily battery of extended tests.
```

Comment:
Isn't this identical to on_daily ?

Reply:
This one just runs if there were changes since yesterday. For instance: we shouldn't run CodeQL everyday if there have been no changes. I'll clarify it in the header.